### PR TITLE
Inital dockerization of the app

### PR DESCRIPTION
**Pull request by mhz-edu**
Backend and db are put into containers. Added docker compose to spin up both.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM node:alpine
```
Comment (irenlian) [inline Dockerfile]: It would be nice to fix a version

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+version: "3.8"
+
+services:
+  postgres:
+    image: postgres:12-alpine
+    container_name: postgres
+    env_file:
+      - ./.env
+    volumes:
+      - data:/var/lib/postgresql/data
```
Comment (irenlian) [inline docker-compose.yaml]: I suppose `container_name` is not required if it is equal to the naming in 6th line.

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  postgres:
+    image: postgres:12-alpine
+    container_name: postgres
+    env_file:
+      - ./.env
+    volumes:
+      - data:/var/lib/postgresql/data
+
+  backend:
+    build: ./.
```
Comment (irenlian) [inline docker-compose.yaml]: Just to check, shouldn't be here `.`?

**Review by irenlian — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (mhz-edu) [reply Dockerfile]: Updated node and postgres versions to match with used on heroku

File: docker-compose.yaml (hunk 2 above)
Comment (mhz-edu) [reply docker-compose.yaml]: If container_name is not set, then on docker-compose up container will be named like <folder_name>_postgres_1

File: docker-compose.yaml (hunk 3 above)
Comment (mhz-edu) [reply docker-compose.yaml]: Fixed.

**Pull request merged**